### PR TITLE
Force libsdl-native being used

### PR DESCRIPTION
**Pull request by floion**

### CODE REVIEW

File: layers/meta-resin-qemu/conf/samples/local.conf.sample [hunk 1]
```diff
@@ -67,6 +67,5 @@ BB_DISKMON_DIRS = "\
 # libsdl library available on your build system.
 PACKAGECONFIG_append_pn-qemu-native = " sdl"
 PACKAGECONFIG_append_pn-nativesdk-qemu = " sdl"
-ASSUME_PROVIDED += "libsdl-native"
```
Comment (agherzan) [inline layers/meta-resin-qemu/conf/samples/local.conf.sample]: I think there was a problem with the native built of this library. Was this fixed in the later versions of poky?

Comment (floion) [inline layers/meta-resin-qemu/conf/samples/local.conf.sample]: Don't know of a problem. What sort of issue?

**Comment by agherzan** [comment]
https://bugzilla.yoctoproject.org/show_bug.cgi?id=6509 - looks good to merge

**Comment by agherzan** [comment]
Builds though is failing.

File: layers/meta-resin-qemu/conf/samples/local.conf.sample (hunk 1 above)
Comment (agherzan) [inline layers/meta-resin-qemu/conf/samples/local.conf.sample]: https://bugzilla.yoctoproject.org/show_bug.cgi?id=6509 - looks good to merge

**Comment by floion** [comment]
Yes, looking into it

**Pull request merged**